packages/core/src/universal/hooks/useSubscription/index.ts

```
@@ -75,10 +104,21 @@
           break;
 
         case 'error':
+          const errorMessage = message.payload?.message || 'Subscription error';
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)